src/main/kotlin/no/nav/syfo/infrastructure/database/repository/VurderingRepository.kt

```diff
@@ -36,18 +42,44 @@ class VurderingRepository(private val database: DatabaseInterface) : IVurderingR
 
         database.connection.use { connection ->
             val pVurdering = connection.createVurdering(vurdering)
-            val pVarsel = connection.createVarsel(
+            connection.createVarsel(
                 vurderingId = pVurdering.id,
                 varsel = vurdering.varsel,
             )
             connection.createPdf(
-                varselId = pVarsel.id,
+                vurderingId = pVurdering.id,
                 pdf = pdf,
             )
             connection.commit()
         }
     }
 
+    override fun update(vurdering: Vurdering) = database.connection.use { connection ->
+        connection.prepareStatement(UPDATE_VURDERING).use {
+            it.setString(1, vurdering.journalpostId)
+            it.setObject(2, nowUTC())
+            it.setString(3, vurdering.uuid.toString())
+            val updated = it.executeUpdate()
+            if (updated != 1) {
+                throw SQLException("Expected a single row to be updated, got update count $updated")
+            }
+        }
+        connection.commit()
+    }
+
+    override fun getNotJournalforteVurderinger(): List<Pair<Vurdering, ByteArray>> =
+        database.connection.use { connection ->
+            connection.prepareStatement(GET_NOT_JOURNALFORT_VURDERING).use {
+                it.executeQuery()
+                    .toList {
+                        Pair(
+                            toPVurdering(),
+                            getBytes("pdf"),
+                        )
+                    }
+            }.map { (pVurdering, pdf) -> Pair(pVurdering.toVurdering(null), pdf) }
```

Comment:
Akkurat på `toVurdering(null)` var vi litt usikre. For å journalføre så er jo egentlig ikke varslet tilknyttet vurderingen noe interessant, men det betyr at man ved uthenting av de ikke-journalførte vurderingene ikke vil ha et komplett/helhetlig `Vurdering`-objekt. Skal vi tilstrebe å ha det? I så fall må vi legge inn det i queryen også.

Reply:
Synes ikke det gjør noe at varsel er null i dette tilfellet.